jwst/resample/resample_utils.py

```diff
@@ -163,7 +163,7 @@ def build_driz_weight(model, weight_type=None, good_bits=None):
     else:
         warnings.warn("weight_type set to None.  Setting drizzle weight map to 1",
                       RuntimeWarning)
-        result = np.ones(model.data.shape, dtype=model.data.dtype)
+        result = np.ones(model.data.shape, dtype=model.data.dtype) * dqmask
```

Comment:
Is `dqmask` here always `1` or `0` (or `True/False`)? Also, shouldn't it have the same shape as `model.data`? If so, wouldn't it be more clear to do `dqmask.astype(model.data.dtype.type)`? (or, of `dqmask` is not `1` or `0` - then (dqmask == 0).astype(model.data.dtype.type)`)?

Reply:
I see on line `148` it is a boolean array => all good.